schema/fair.js

```diff
@@ -4,23 +4,29 @@ import date from './fields/date';
 import Profile from './profile';
 import Image from './image';
 import Location from './location';
+import { GravityIDFields } from './object_identification';
 import {
   GraphQLObjectType,
+  GraphQLID,
   GraphQLString,
   GraphQLBoolean,
   GraphQLNonNull,
 } from 'graphql';
 
+const OrganizerType = new GraphQLObjectType({
```

Comment:
wonder if we should be more specific about it being a `FairOrganizer` (https://github.com/artsy/gravity/blob/2fbb850affa5b20d066f4a72c33c61e0d1732c19/app/models/domain/fair_organizer.rb)